Doc/tutorial/introduction.rst

```diff
@@ -478,73 +478,118 @@ example::
 First Steps Towards Programming
 ===============================
 
-Of course, we can use Python for more complicated tasks than adding two and two
-together.  For instance, we can write an initial sub-sequence of the
-`Fibonacci series <https://en.wikipedia.org/wiki/Fibonacci_number>`_
-as follows::
-
-   >>> # Fibonacci series:
-   ... # the sum of two elements defines the next
-   ... a, b = 0, 1
-   >>> while a < 10:
-   ...     print(a)
-   ...     a, b = b, a+b
-   ...
+We can use Python for more complex tasks than adding two and two together.
+For instance, we can compute the numbers of the
+`Fibonacci series <https://en.wikipedia.org/wiki/Fibonacci_number>`_.
+To do that we need to utilize three following concepts.
+
+Multiple Assignment
+-------------------
+
+A *multiple assignment* of variables us allows to set values of more than one
+variable on a single line.  Notice that if the value assignment is a
+expression, it is first evaluated and then assigned.::
+
+   >>> a, b = 1, 5  # multiple assignment of two variables
+   >>> a
    0
-   1
+   >>> b
+   5
+   >>> a, b = b, a + b  # first a is set to value of b, then b is set to sum of a and b
+   >>> a
+   5
+   >>> b
+   6
+
+Repeating our code
+------------------
+
+The :keyword:`while` loop is example of a repeating cycle.  It performs
+a block of code over as long as the condition (in example: ``a < 10``) is
```

Comment:
"in example" > "for example"

Reply:
What I wanted to say is that "in the example below the condition is the following expression `a < 10`". I will rework that.